docs/source/advanced/language-models.rst

```diff
@@ -42,19 +42,19 @@ Prepare Text Graph Data
 Create Raw ACM Graph Tables
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This tutorial will use the same ACM data as the :ref:`Use Your Own Data<use-own-data>` tutorial but add text data as node features.
+This tutorial will use the same ACM data as the :ref:`Use Your Own Data<use-own-data>` tutorial but add text data as node features. This tutorial assume users have cloned GraphStorm source code into the ``<path-to-graphstorm>`` folder.
 
-First go the ``/graphstorm/examples/`` folder.
+First go the ``<path-to-graphstorm>`` folder.
 
 .. code-block:: bash
 
-    cd /graphstorm/examples 
+    cd <path-to-graphstorm>
```

Comment:
Instead of this, let's have the first couple of lines in every tutorial be

```
git clone https://github.com/awslabs/graphstorm.git
cd graphstorm
```

That way we ensure the reader can directly copy/paste the entire code section of the tutorial and they should work without any modification from the user.

Reply:
Updated with git clone and cd. thanks